gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -181,8 +181,8 @@ const CssVariablesPage = () => {
           <PropertyDocItem propertyVar="--nx-border-default">
             Standard border styles for higher-level elements.
           </PropertyDocItem>
-          <PropertyDocItem propertyVar="--nx-border-subsection">
```

Comment:
Usually when we deprecate a variable we don't completely remove its docs, we rewrite them to explain that its deprecated, often using a warning alert.  See `NxModal` `onClose`

Reply:
> Usually when we deprecate something we don't completely remove its docs, we rewrite them to explain that its deprecated, often using a warning alert. See `NxModal` `onClose`

Right: https://github.com/sonatype/sonatype-react-shared-components/pull/444/commits/b4683e8f1bf6bd06abe8563034dc2552e93f5f08